tests/unit/cache/test_localfs.py

```diff
@@ -137,15 +132,12 @@ def test_fetch_success(self):
         # Create a temporary cache dir
         tmp_dir = tempfile.mkdtemp(dir=RUNTIME_VARS.TMP)
 
-        # Create a new serializer object to use in function patches
-        serializer = salt.payload.Serial(self)
-
         # Use the helper function to create the cache file using localfs.store()
-        self._create_tmp_cache_file(tmp_dir, serializer)
+        self._create_tmp_cache_file(tmp_dir)
 
         # Now fetch the data from the new cache key file
         with patch.dict(localfs.__opts__, {"cachedir": tmp_dir}):
-            with patch.dict(localfs.__context__, {"serial": serializer}):
+            with patch.dict(localfs.__context__, {"serial": salt.payload}):
```

Comment:
Drop `__context__["serial"]` too from localfs?

Reply:
Thought I tackled this already, maybe I missed this one?

Reply:
Probably a missed one.